pkgs/development/rocm-modules/6/composable_kernel/base.nix

```diff
@@ -0,0 +1,169 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rocmUpdateScript,
+  cmake,
+  rocm-cmake,
+  rocm-merged-llvm,
+  clr,
+  rocminfo,
+  hipify,
+  gitMinimal,
+  gtest,
+  zstd,
+  buildTests ? false,
+  buildExamples ? false,
+  gpuTargets ? (
+    clr.localGpuTargets or [
+      "gfx900"
+      "gfx906"
+      "gfx908"
+      "gfx90a"
+      "gfx942"
+      "gfx1030"
+      "gfx1100"
+      "gfx1101"
+      "gfx1102"
```

Comment:
Should `gfx1200` and `gfx1201` be there too ? ( RX 9xxx (XT) )
I can detect `gfx1201` with rocminfo from this PR.
Not luck having ollama detect it though.

Thanks for the awesome work !!

Reply:
I think we have to wait until ROCm 6.4 for gfx1200 support

Reply:
Here is the ROCm PR for refence :)
https://github.com/ROCm/ROCm/pull/4162/files#diff-1008abbfd7af74f71debc6959e6b4763ad7ed29c73402dc86e6513893a73c407